src/mem/largealloc.h

```diff
@@ -193,7 +193,7 @@ namespace snmalloc
         {
           // The first page is already in "use" for the stack element,
           // this will need zeroing for a YesZero call.
-          if (zero_mem == YesZero)
+          if constexpr (zero_mem == YesZero)

```

Comment:
We should probably do the same for the `if (allow_reserve == YesReserve)` check above (line 141)